packages/cardano-services/README.md

```diff
@@ -1,7 +1,92 @@
 # Cardano JS SDK | Cardano GraphQL Services
+Libraries and program entrypoints for services that facilitate remote data and submit access using 
+[Provider] interfaces over HTTP, with _optional_ queue-based transaction submission; The 
+[TxSubmitHttpService] can be configured to submit directly via [Ogmios], or via a RabbitMQ broker,
+with one or more workers handling submission and response job creation.
+
+## Features
+- [CLI] or run scripts for the [HTTP server](./src/run.ts) and [worker](./src/startWorker.ts) with 
+configuration via environment variables and _optional_ loading of secrets from disk.
+- Service port discovery via DNS resolution, or static configuration.
+- Fault-tolerant transaction submission via persistent queue, or direct submission.
+- _Optional_ [Prometheus] metrics available at `/metrics`
+
+## Services
+### HTTP Server
+The HTTP server can be started with one or more provider modules by name, segmented by URL path. 
+Run the [CLI] with `start-server --help` to see the full list of options, or inspect the 
+environment variables within [./src/run.ts](./src/run.ts). 
+
+### Worker
+A worker must be started when opting for queue-based transaction submission.
+Run the [CLI] with `start-worker --help` to see the full list of options, or inspect the
+environment variables within [./src/startWorker.ts](./src/startWorker.ts).
+
+
+## Examples
+
+_The following examples require the [install and build] steps 
+to be completed._
+
+#### All Providers | Static Service Config | Direct Tx Submission
+
+- The server will expose all [Provider] HTTP services
+- Transactions will be submitted directly via [Ogmios], running at `ws://localhost:1338`.
+- Connects to [PostgreSQL] service running at `localhost:5432`
+- HTTP API exposed using a custom API URL
+
+``` console
+./dist/cli.js \
+  start-server \
+    --api-url http://localhost:6000 \
+    --cardano-node-config-path ./config/cardano-node/config.json \
+    --ogmios-url  ws://localhost:1338 \
```

Comment:
Shouldn't we use in the sample port `1337` as the `docker-compose` says [here](https://github.com/input-output-hk/cardano-js-sdk/blob/b64fd99daffa96f14873992379aba0fa77b2e272/packages/cardano-services/docker-compose.yml#L39-L40) ?

Reply:
That would be confusing as it would then not be required given `ws://localhost:1337` is the default